compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt

```diff
@@ -91,7 +91,15 @@ internal class ComposeAccessible(
 
     val composeAccessibleContext: ComposeAccessibleComponent by lazy { ComposeAccessibleComponent() }
 
-    override fun getAccessibleContext(): AccessibleContext {
+    var removed = false
```

Comment:
Am I right that tree will be recreated after a while and ComposeAccessible will be collected? Otherwise it looks like a leak

Reply:
Not sure I understand what you mean. The node is marked as `removed=false` when it's no longer in the tree. I'm not storing it anywhere it wasn't before.

`sun.lwawt.macosx.CAccessible` will probably keep storing a reference to it, but only until some other node becomes "active" (it only stores one `Accessible`).

Reply:
Yes, I am talking about `sun.lwawt.macosx.CAccessible`, so I hope that ComposeAccessible won't be stored there forever